quiver/benchmark_extraction.py

```diff
@@ -8,7 +8,7 @@
 import re
 from typing import Any, Dict, List, Union
 
-METS = '{http://www.loc.gov/METS/}'
+from .constants import *
```

Comment:
Always do explicit imports, it's more verbose but makes it clear where all the names come from without an IDE.